### PR TITLE
Logan2

### PR DESCRIPTION
**Pull request by LoganFinnegan**

### CODE REVIEW

File: spec/game_list_spec.rb [hunk 1]
```diff
@@ -1,38 +1,46 @@
 require 'spec_helper'
 
 RSpec.describe GameList do
-  it "can create a new Game List class instance" do
-    game_list = GameList.new('./data/games_subset.csv', 'stat_tracker')
+  before(:each) do
+    locations = {
```
Comment (megroth2) [inline spec/game_list_spec.rb:5]: This is a minor tweak, but for these specs I don't think we have to include all 3 keys for the locations hash - so in this case I think we only need games and we could reduce this to just:
```
locations = {
      games: "./data/games_subset.csv"
    }
```

File: spec/team_list_spec.rb [hunk 2]
```diff
@@ -2,19 +2,27 @@
 
 RSpec.describe TeamList do
   before(:each) do
-    @new_teamlist = TeamList.new("./data/teams_subset.csv", StatTracker )
+    locations = {
```
Comment (megroth2) [inline spec/team_list_spec.rb:5]: same as above, I think we can shorten this to just:
```
    locations = {
      teams: "./data/teams_subset.csv"
    }
```

File: spec/team_list_spec.rb [hunk 3]
```diff
@@ -2,19 +2,27 @@
...
   end
 
   it 'can create teams' do
-    @new_teamlist.create_teams("./data/teams_subset.csv")
-    
-    expect(@new_teamlist.teams.count).to eq(20)
+    @team_list.create_teams("./data/teams_subset.csv")
+    #this will break ousdie of dummy files
+    # expect(@team_list.teams.count).to eq(20)
+    expect(@team_list.teams).to all(be_an_instance_of Team)
```
Comment (megroth2) [inline spec/team_list_spec.rb:26]: Cool, agree this is a better test than counting

File: spec/game_list_spec.rb (hunk 1 above)
Comment (s2an) [reply spec/game_list_spec.rb:5]: I second this. Only add what you need :)

Comment (LoganFinnegan) [reply spec/game_list_spec.rb:5]: We want to have all three because some of the later methods need access to all three csv files. Currently the methods dont need them but they would if we got that far.
